src/index.ts

```diff
@@ -245,11 +249,19 @@ async function deployService(serviceName: ServiceName, app: AppConfig) {
     ? ["deploy"]
     : ["versions", "upload", "--experimental-versions"];
 
-  return execa(
-    "npx",
-    ["wrangler", ...command, "--config", "./wrangler.toml", ...extraArgs],
-    options,
-  ).pipeStdout!(process.stdout);
+  const deployArgs = [
```

Comment:
Extra args won't be added. Leave the return as is, just update extra args before